### PR TITLE
Raven db 4845

### PR DESCRIPTION
**Pull request by maximburyak**
Fixed problem with ForceReadFromMaster being ignored
When reading from all in cluster, the round robin mechanism is not working

### CODE REVIEW

**Comment by ppekrol** [comment]
@maximburyak Tests could not be run, because solution stops compiling after this PR (missing files). Please rebase and force-push.

File: Raven.Client.Lightweight/Connection/Request/IRequestExecuter.cs [hunk 1]
```diff
@@ -25,5 +25,6 @@ public interface IRequestExecuter
         event EventHandler<FailoverStatusChangedEventArgs> FailoverStatusChanged;
 
         void AddHeaders(HttpJsonRequest httpJsonRequest, AsyncServerClient serverClient, string currentUrl);
+        void SetReadStripingBase(int strippingBase);
```
Comment (ayende) [inline Raven.Client.Lightweight/Connection/Request/IRequestExecuter.cs:28]: No one is calling this, why does this exists?

Comment (ayende) [inline Raven.Client.Lightweight/Connection/Request/IRequestExecuter.cs:28]: Nevermind, saw that in the other commit

**Comment by ayende** [comment]
``````
CSC : error CS2001: Source file 'C:\Work\ravendb-3.5\Raven.Tests.Raft\ClusterDatabaseIssue.cs' could not be found. [C:\Work\ravendb-3.5\Raven.Tests.Raft\Rav
en.Tests.Raft.csproj]```
``````

**Pull request closed without merging**